setzer/dialogs/build_save/build_save.py

```diff
@@ -44,8 +44,8 @@ def run(self, document):
     def setup(self, document):
         self.view = Gtk.MessageDialog(self.main_window, 0, Gtk.MessageType.QUESTION)
 
-        self.view.set_property('text', 'Document »' + document.get_displayname() + '« has no filename.')
-        self.view.format_secondary_markup('Please save your document to a file, so the build system knows where to put the .pdf (it will be in the same folder as your document).')
+        self.view.set_property('text', _('Document »{document}« has no filename.').format(document=document.get_displayname()))
```

Comment:
This is a better way than using a '+' in strings.

Reply:
Agreed